app/main.py

```diff
@@ -1,2 +1,54 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, value: Any) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+
+
 class Dictionary:
-    pass
+    def __init__(self, default_capacity: int = 8,
+                 load_factor: float = 0.66) -> None:
+        self.capacity = default_capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self._data = [None] * self.capacity
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
```

Please fix everywhere